backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/payload/ShareConnectedModePayload.java

```diff
@@ -25,5 +25,6 @@
 public record ShareConnectedModePayload(@SerializedName("manual_bindings_count") @Nullable Integer manualAddedBindingsCount,
                                         @SerializedName("imported_bindings_count") @Nullable Integer importedAddedBindingsCount,
                                         @SerializedName("auto_bindings_count") @Nullable Integer autoAddedBindingsCount,
-                                        @SerializedName("exported_connected_mode_count") @Nullable Integer exportedConnectedModeCount) {
+                                        @SerializedName("exported_connected_mode_count") @Nullable Integer exportedConnectedModeCount,
+                                        @SerializedName("suggested_remote_bindings_count") @Nullable Integer suggestedRemoteBindingsCount) {
```

Comment:
I think you don't need this one, this is the old way to measure things. I see you modified  `TelemetryMeasuresBuilder` already so you should be good